### PR TITLE
Update "paymentInitiation_json" model and descriptions

### LINKED ISSUE #31: Description of transactionCurrency
In the description for the Model **paymentInitiation_json** we have a table which defines for each property, whether it is mandatory or not (depending on the payment type).

While the name of the property in the model is **`transactionCurrency`** the table include this property as **`currencyOfTransfer`**.  Please rename the name in the table from **`currencyOfTransfer`** to **`transactionCurrency`** (do not rename the property).

The **`currencyOfTransfer`** references Footnote 1) in some columns. There is also a footnote 7) whose text refers to this property but it is never used. I don't know which footnote would be correct for this property: 1? 7? 1 AND 7?

If 7) is not used, then please remove it. If it is used then rename the property in the footnote too.

**NOTE: There is a corresponding table in periodic payments and bulk payments. Please apply the above corrections to ALL 3 tables and footnotes.**

### LINKED ISSUE #32: Remove non applicable properties
In the table which describes all properties for each payment type two properties are **n.a.** for **each** payment type:

- remittanceInformationUnstructuredArray
- requestedExecutionTime

Why do we even mention them if they are never used? I suggest, we remove them.

**NOTE: There is a corresponding table in periodic payments and bulk payments. Please apply the above corrections to ALL 3 tables and footnotes.**

### LINKED ISSUE #43: Missing property `creditorAgentName` in single payment
In the table which describes all properties for each payment type there is a property `creditorAgentName` which is missing in the model paymentInitiation_json (it is commented, why?). According to the description of this property, it is a property which should be around.

We should add (uncomment) this property to the single payment model.

### LINKED ISSUE #44: Missing description for property `intermediaryAgent` in single payment
The property `intermediaryAgent` exists in the model `paymentInitiation_json` but is not documented in the descritpion of this model.

Is this a property which should be removed or should the description be completed?

### PR DESCRIPTION
**Pull request by wishopt**
This pull request adresses the following issues: #31,  #32,  #33, #43, #44

Please note that only the "paymentInitiation_json" model was changed. "periodicPaymentInitiation_json" and "bulkPaymentInitiation_json" will be updated after this pull request has been discussed, as they are based on the single payment model.

### CODE REVIEW

File: swiss-ng-api.yaml [hunk 1]
```diff
@@ -5975,6 +5978,11 @@ components:
         endToEndIdentification:
           type: string
           maxLength: 35
+#        instructionIdentification: # N.A. for all, but can be used by ASPSP if needed
+#          type: string
+#          maxLength: 35
+#        debtorName: # N.A. for all, but can be used by ASPSP if needed
+#          $ref: "#/components/schemas/debtorName"
```
Comment (yves-bussard) [inline swiss-ng-api.yaml]: debtorName is mandatory for all. See line 5921 (new file) or 5920 (old file). These lines should NOT be in comments.

Here you are editing the properties of the model paymentInitiation_json (line 5890). Make sure you are editing the properties of the correct model.

File: swiss-ng-api.yaml [hunk 2]
```diff
@@ -6015,13 +6023,15 @@ components:
...
         remittanceInformationStructured: # N.A. for all, but can be used by ASPSP if needed
           $ref: "#/components/schemas/remittanceInformationStructured"
-        requestedExecutionDate: # N.A. for all, but can be used by ASPSP if needed
-          type: string
-          format: date
+#       remittanceInformationStructuredArray: # N.A. for all, but can be used by ASPSP if needed
+#         $ref: "#/components/schemas/remittanceInformationStructuredArray"
+#       requestedExecutionDate: # N.A. for all, but can be used by ASPSP if needed
+#         type: string
+#         format: date
```
Comment (yves-bussard) [inline swiss-ng-api.yaml]: The **requestedExecutionDate** is actually mandatory for all. but the **requestedExecutionTime** is not and should be commented.

**Comment by yves-bussard** [comment]
#33 referred to periodic payments but you changed it for bulk payments. This is ok, but does not resolve #33.

**Review by yves-bussard — CHANGES_REQUESTED** [review]

File: swiss-ng-api.yaml (hunk 1 above)
Comment (blehers) [reply swiss-ng-api.yaml]: You're right, we'll be corrected.

File: swiss-ng-api.yaml (hunk 2 above)
Comment (blehers) [reply swiss-ng-api.yaml]: You're right, we'll be corrected.

**Pull request merged**